### DISCUSSION TITLE
JsonIgnore and other custom attributes

### DISCUSSION THREAD

**Discussion by czirok** [CommunityToolkit/dotnet#228] [Q&A] [answered]
It is possible to add

[ObservableProperty]
[JsonIgnore] **<- this**
[MyOtherAttribute] **<- this**
bool _isValid;

[ObservableProperty]
[JsonIgnore] **<- this**
[MyOtherAttribute2] **<- this**
string _other;

to the auto-generated source code?

Thank you.

**Comment by michael-hawker** [comment, marked as answer]
Duplicate of #217

See issue #208
